CoseSignTool/CoseCommand.cs

```diff
@@ -136,7 +136,7 @@ protected ExitCode TryGetStreamFromPipeOrFile(FileInfo? file, string optionName,
             {
                 content = UseAdvancedStreamHandling ? file.GetStreamResilient(MaxWaitTime) : file.GetStreamBasic(MaxWaitTime);
                 return
-                    content.IsNullOrEmpty() ?
+                    content.IsNullOrEmpty(10000) ?

```

Comment:
If we default to 10seconds, we should have an argument to downgrade this for external customers who might have empty files on low stress systems.

Reply:
Adding a "FailFast" switch to account for this